db/schema.rb

```diff
@@ -10,7 +10,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2019_12_31_232856) do
+ActiveRecord::Schema.define(version: 2020_03_04_181505) do
```

Comment:
not sure why this schema change is in this branch -- we need to verify this is correct.